Header header
float64[] mean
float64[] sigma
